CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ project(eosio_contracts)
 set(VERSION_MAJOR 3)
 set(VERSION_MINOR 2)
 set(VERSION_PATCH 0)
-set(VERSION_SUFFIX dev)
+set(VERSION_SUFFIX rc1)
```

Comment:
This is fine with me as is. But did you want to bump the version in `main` to `3.3.0-dev` or keep it in sync with the latest release branch for now?